frontend/kesaseteli/employer/src/types/application.d.ts

```diff
@@ -0,0 +1,12 @@
+import Company from 'shared/types/company';
+
+type Application = {
+  id: string;
+  company: Company;
+  invoicer_email: string;
+  invoicer_name: string;
+  invoicer_phone_number: string;
+  status: string;
```

Comment:
Could the `status` type be more specific?

Reply:
Could be, I didnt know the values while the time of writing the code but now I know. I can fix it.